### PR TITLE
Add plugin Auto Login Local Configuration

### PR DESCRIPTION
**Pull request by cp2004**
Latest plugin, ready for review. 🙂 

Hopefully everything is good, I have tested the build locally.

### CODE REVIEW

File: _plugins/autologin_config.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  - accesscontrol
+
+screenshots:
+  - url: /assets/img/plugins/autologin_config/instructions.png
+    alt: instructions
+    caption: Instructions for configuration
+  - url: /assets/img/plugins/autologin_config/config.png
+    alt: configuration
+    caption: Configuration UI
+  - ...
```
Comment (jneilliii) [inline _plugins/autologin_config.md:30]: remove this

**Review by jneilliii — CHANGES_REQUESTED** [review]
upon initial review of the failed checks these items need to be addressed.

File: _plugins/autologin_config.md (hunk 1 above)
Comment (jneilliii) [reply _plugins/autologin_config.md:30]: line 39

**Comment by jneilliii** [comment]
I'm not sure of the failed build check relative to authors @foosel, recommendations?

File: _plugins/autologin_config.md (hunk 1 above)
Comment (cp2004) [reply _plugins/autologin_config.md]: Fixed

**Comment by cp2004** [comment]
Yeah, the authors confused me, since there's just one of me. Fixed the screenshots. Would be great if I could set about clones fixing bugs, maybe GH Actions knows something I don't...

**Comment by jneilliii** [comment]
Looks like that fixed all the build checks, thanks. I will continue the review process now.

**Review by jneilliii — APPROVED** [review]

**Pull request merged**